apps/internal-ui/src/components/Input/InputPassword/InputPasswordRules.tsx

```diff
@@ -3,15 +3,15 @@ import clsx from 'clsx';
 import { Icon } from '@/components/Icon';
 import { InputPasswordRuleProps } from '@/components/Input/InputPassword/types';
 
-const InputPasswordRules = ({ rules, value = '' }: InputPasswordRuleProps) => {
+const InputPasswordRules = ({ rules, value }: InputPasswordRuleProps) => {
   return (
     <ul className='shadow-in-4 rounded-in-8 bg-in-white flex flex-col gap-y-0.5 px-2.5 py-2'>
       {Object.entries(rules).map(([key, { message, regex }]) => (
         <li
           key={key}
           className={clsx(
             'text-in-body-12-m flex gap-x-0.5',
-            regex.test(value) ? 'text-in-green-05' : 'text-in-gray-05',
+            regex.test(value ?? '') ? 'text-in-green-05' : 'text-in-gray-05',
```

Comment:
p3: value에 대한 판별은 바깥으로 빼면 test메소드 실행까지 안 해도 될 것 같은데 어떨까요?

```suggestion
          !!value && regex.test(value) ? 'text-in-green-05' : 'text-in-gray-05',
```